src/allmydata/frontends/magic_folder.py

```diff
@@ -33,6 +33,8 @@ def get_inotify_module():
             from allmydata.windows import inotify
         elif runtime.platform.supportsINotify():
             from twisted.internet import inotify
+        elif sys.platform != "linux":
+            from allmydata.watchdog import inotify
```

Comment:
watchdog supports nearly all platforms. what do you want this conditional to look like instead?